<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>550</width>
    <height>440</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>550</width>
    <height>440</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>550</width>
    <height>440</height>
   </size>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::NoContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string notr="true">v.1.1</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../res/Res.qrc">
    <normaloff>:/images/images/application-icon.png</normaloff>:/images/images/application-icon.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="contextMenuPolicy">
    <enum>Qt::NoContextMenu</enum>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget  {
	background-color: rgb(230, 230, 230);
}</string>
   </property>
   <widget class="QFrame" name="frame5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>370</y>
      <width>530</width>
      <height>60</height>
     </rect>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::NoContextMenu</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QFrame {
	background-color: rgb(190, 190, 190);
	
	border: 1px solid rgb(0, 0, 0);
	border-radius: 2px;

}
</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="lineWidth">
     <number>5</number>
    </property>
    <property name="midLineWidth">
     <number>2</number>
    </property>
    <widget class="QPushButton" name="settingsButton">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>230</width>
       <height>40</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::StrongFocus</enum>
     </property>
     <property name="contextMenuPolicy">
      <enum>Qt::NoContextMenu</enum>
     </property>
     <property name="toolTipDuration">
      <number>1</number>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
	font: 75 12pt &quot;Liberation Sans&quot;;
	color: rgb(255, 255, 255);

	background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 0.67,  stop: 0 rgb(50, 80, 150), stop: 1 rgb(10, 20, 100));

	border: 1px solid  rgb(0, 0, 0);
	border-radius: 2px;
}

QPushButton:hover {
	background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 0.67,  stop: 0 rgb(60, 90, 220), stop: 1 rgb(20, 30, 150));
}

QPushButton:disabled {
	background-color: rgb(150, 150, 180);
}
</string>
     </property>
     <property name="text">
      <string>SETTINGS</string>
     </property>
    </widget>
    <widget class="QPushButton" name="exitButton">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>10</y>
       <width>230</width>
       <height>40</height>
      </rect>
     </property>
     <property name="tabletTracking">
      <bool>false</bool>
     </property>
     <property name="focusPolicy">
      <enum>Qt::StrongFocus</enum>
     </property>
     <property name="contextMenuPolicy">
      <enum>Qt::NoContextMenu</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
	font: 75 12pt &quot;Liberation Sans&quot;;
	color: rgb(255, 255, 255);

	background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 0.67,  stop: 0 rgb(180, 0, 0), stop: 1 rgb(100, 0, 0));

	border: 1px solid  rgb(0, 0, 0);
	border-radius: 2px;
}

QPushButton:hover {
	background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 0.67,  stop: 0 rgb(220, 0, 0), stop: 1 rgb(130, 0, 0));
}

QPushButton:disabled {
	background-color: rgb(180, 150, 150);
}
</string>
     </property>
     <property name="text">
      <string>EXIT</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>530</width>
      <height>110</height>
     </rect>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::NoContextMenu</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QFrame {
	background-color: rgb(190, 190, 190);
	
	border: 1px solid  rgb(0, 0, 0);
	border-radius: 2px;
}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="lineWidth">
     <number>5</number>
    </property>
    <property name="midLineWidth">
     <number>2</number>
    </property>
    <widget class="QPushButton" name="bttButton">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>10</y>
       <width>230</width>
       <height>40</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::StrongFocus</enum>
     </property>
     <property name="contextMenuPolicy">
      <enum>Qt::NoContextMenu</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
	font: 75 12pt &quot;Liberation Sans&quot;;
	color: rgb(255, 255, 255);

	background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 0.67,  stop: 0 rgb(150,150, 150), stop: 1 rgb(100, 100, 100));

	border: 1px solid  rgb(0, 0, 0);
	border-radius: 2px;
}

QPushButton:hover {
	background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 0.67,  stop: 0 rgb(180,180, 180), stop: 1 rgb(120, 120, 120));
}

QPushButton:disabled {
	background-color: rgb(150, 150, 150);
}
</string>
     </property>
     <property name="text">
      <string>LOC FILE TO TEXT FILE</string>
     </property>
    </widget>
    <widget class="QPushButton" name="ttbButton">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>60</y>
       <width>230</width>
       <height>40</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::StrongFocus</enum>
     </property>
     <property name="contextMenuPolicy">
      <enum>Qt::NoContextMenu</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
	font: 75 12pt &quot;Liberation Sans&quot;;
	color: rgb(255, 255, 255);

	background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 0.67,  stop: 0 rgb(150,150, 150), stop: 1 rgb(100, 100, 100));

	border: 1px solid  rgb(0, 0, 0);
	border-radius: 2px;
}

QPushButton:hover {
	background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 0.67,  stop: 0 rgb(180,180, 180), stop: 1 rgb(120, 120, 120));
}

QPushButton:disabled {
	background-color: rgb(150, 150, 150);
}
</string>
     </property>
     <property name="text">
      <string>TEXT FILE TO LOC FILE</string>
     </property>
    </widget>
    <widget class="QLabel" name="label1">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>270</width>
       <height>90</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
	font: 10pt &quot;Liberation Sans&quot;;
	color: rgb(20, 20, 20);

	
	background-color: rgb(255, 255, 255);

	border: 1px solid  rgb(0, 0, 0);
	border-radius: 2px;
}
</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="lineWidth">
      <number>5</number>
     </property>
     <property name="midLineWidth">
      <number>2</number>
     </property>
     <property name="text">
      <string>Convert binary .loc file to text file and back. Default file names and processing directory may be set in settings or choosed manually in choose files window.</string>
     </property>
     <property name="textFormat">
      <enum>Qt::PlainText</enum>
     </property>
     <property name="alignment">
      <set>Qt::AlignHCenter|Qt::AlignTop</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
     <property name="margin">
      <number>5</number>
     </property>
     <property name="textInteractionFlags">
      <set>Qt::NoTextInteraction</set>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>530</width>
      <height>110</height>
     </rect>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::NoContextMenu</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QFrame {
	background-color: rgb(190, 190, 190);
	
	border: 1px solid  rgb(0, 0, 0);
	border-radius: 2px;
}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="lineWidth">
     <number>5</number>
    </property>
    <property name="midLineWidth">
     <number>2</number>
    </property>
    <widget class="QPushButton" name="mbButton">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>10</y>
       <width>230</width>
       <height>40</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::StrongFocus</enum>
     </property>
     <property name="contextMenuPolicy">
      <enum>Qt::NoContextMenu</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
	font: 75 12pt &quot;Liberation Sans&quot;;
	color: rgb(255, 255, 255);

	background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 0.67,  stop: 0 rgb(150,150, 150), stop: 1 rgb(100, 100, 100));

	border: 1px solid  rgb(0, 0, 0);
	border-radius: 2px;
}

QPushButton:hover {
	background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 0.67,  stop: 0 rgb(180,180, 180), stop: 1 rgb(120, 120, 120));
}

QPushButton:disabled {
	background-color: rgb(150, 150, 150);
}
</string>
     </property>
     <property name="text">
      <string>MERGE TWO LOC FILES</string>
     </property>
    </widget>
    <widget class="QPushButton" name="mtButton">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>60</y>
       <width>230</width>
       <height>40</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::StrongFocus</enum>
     </property>
     <property name="contextMenuPolicy">
      <enum>Qt::NoContextMenu</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
	font: 75 12pt &quot;Liberation Sans&quot;;
	color: rgb(255, 255, 255);

	background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 0.67,  stop: 0 rgb(150,150, 150), stop: 1 rgb(100, 100, 100));

	border: 1px solid  rgb(0, 0, 0);
	border-radius: 2px;
}

QPushButton:hover {
	background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 0.67,  stop: 0 rgb(180,180, 180), stop: 1 rgb(120, 120, 120));
}

QPushButton:disabled {
	background-color: rgb(150, 150, 150);
}
</string>
     </property>
     <property name="text">
      <string>MERGE TWO TEXT FILES</string>
     </property>
    </widget>
    <widget class="QLabel" name="label2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>270</width>
       <height>90</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
	font: 10pt &quot;Liberation Sans&quot;;
	color: rgb(20, 20, 20);

	
	background-color: rgb(255, 255, 255);

	border: 1px solid  rgb(0, 0, 0);
	border-radius: 2px;
}
</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="lineWidth">
      <number>5</number>
     </property>
     <property name="midLineWidth">
      <number>2</number>
     </property>
     <property name="text">
      <string>Merge text or binary .loc files. First file must have translation, second - original file. Merge will be replace original string to translated string and save it to original file.</string>
     </property>
     <property name="textFormat">
      <enum>Qt::PlainText</enum>
     </property>
     <property name="alignment">
      <set>Qt::AlignHCenter|Qt::AlignTop</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
     <property name="margin">
      <number>5</number>
     </property>
     <property name="textInteractionFlags">
      <set>Qt::NoTextInteraction</set>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>250</y>
      <width>530</width>
      <height>110</height>
     </rect>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::NoContextMenu</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QFrame {
	background-color: rgb(190, 190, 190);
	
	border: 1px solid rgb(0, 0, 0);
	border-radius: 2px;

}
</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="lineWidth">
     <number>5</number>
    </property>
    <property name="midLineWidth">
     <number>2</number>
    </property>
    <widget class="QPushButton" name="ftsButton">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>10</y>
       <width>230</width>
       <height>40</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::StrongFocus</enum>
     </property>
     <property name="contextMenuPolicy">
      <enum>Qt::NoContextMenu</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
	font: 75 12pt &quot;Liberation Sans&quot;;
	color: rgb(255, 255, 255);

	background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 0.67,  stop: 0 rgb(150,150, 150), stop: 1 rgb(100, 100, 100));

	border: 1px solid  rgb(0, 0, 0);
	border-radius: 2px;
}

QPushButton:hover {
	background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 0.67,  stop: 0 rgb(180,180, 180), stop: 1 rgb(120, 120, 120));
}

QPushButton:disabled {
	background-color: rgb(150, 150, 150);
}
</string>
     </property>
     <property name="text">
      <string>FILL FILE TO SIZE</string>
     </property>
    </widget>
    <widget class="QLabel" name="label3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>270</width>
       <height>90</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
	font: 10pt &quot;Liberation Sans&quot;;
	color: rgb(20, 20, 20);

	
	background-color: rgb(255, 255, 255);

	border: 1px solid  rgb(0, 0, 0);
	border-radius: 2px;
}
</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="lineWidth">
      <number>5</number>
     </property>
     <property name="midLineWidth">
      <number>2</number>
     </property>
     <property name="text">
      <string>Resize second file to first file size. Added space will be filling by zero bytes. If second file size larger then first file size - nothing will happen.</string>
     </property>
     <property name="textFormat">
      <enum>Qt::PlainText</enum>
     </property>
     <property name="alignment">
      <set>Qt::AlignHCenter|Qt::AlignTop</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
     <property name="margin">
      <number>5</number>
     </property>
     <property name="textInteractionFlags">
      <set>Qt::NoTextInteraction</set>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="glass">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>550</width>
      <height>440</height>
     </rect>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::NoContextMenu</enum>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QFrame {
background-color: rgba(0, 0, 0, 200);
}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>bttButton</tabstop>
  <tabstop>ttbButton</tabstop>
  <tabstop>mbButton</tabstop>
  <tabstop>mtButton</tabstop>
  <tabstop>settingsButton</tabstop>
  <tabstop>exitButton</tabstop>
 </tabstops>
 <resources>
  <include location="../../res/Res.qrc"/>
 </resources>
 <connections/>
 <slots>
  <slot>onButtonClick()</slot>
 </slots>
</ui>
